code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -12,6 +12,130 @@
 	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
 	strip_mod = 0.9
 
+/obj/item/clothing/gloves/fingerless/pugilist
+	name = "armwraps"
+	desc = "A series of armwraps. Makes you pretty keen to start punching people."
+	icon_state = "armwraps"
+	item_state = "armwraps"
+	body_parts_covered = ARMS
+	cold_protection = ARMS
+	strip_delay = 300 //you can't just yank them off
+	var/enhancement = 0 //it's a +0 to your punches because it isn't magical
+	var/inherited_trait = TRAIT_NOGUNS //what are you, dishonoroable?
+	var/secondary_trait = TRAIT_FEARLESS //what are you, a coward?
+
+/obj/item/clothing/gloves/fingerless/pugilist/equipped(mob/user, slot)
+	..()
+	if(slot == SLOT_GLOVES)
+		if(ishuman(user))
+			var/mob/living/carbon/human/H = user
+			ADD_TRAIT(H, TRAIT_PUGILIST, GLOVE_TRAIT)
+			ADD_TRAIT(H, inherited_trait, GLOVE_TRAIT)
+			ADD_TRAIT(H, secondary_trait, GLOVE_TRAIT)
+			H.dna.species.punchdamagehigh += enhancement
+			H.dna.species.punchdamagelow += enhancement
+
+/obj/item/clothing/gloves/fingerless/pugilist/dropped(mob/user)
+	..()
+	REMOVE_TRAIT(user, TRAIT_PUGILIST, GLOVE_TRAIT)
+	REMOVE_TRAIT(user, inherited_trait, GLOVE_TRAIT)
+	REMOVE_TRAIT(user, secondary_trait, GLOVE_TRAIT)
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		H.dna.species.punchdamagehigh = initial(H.dna.species.punchdamagehigh)
+		H.dna.species.punchdamagelow = initial(H.dna.species.punchdamagelow)
+
+/obj/item/clothing/gloves/fingerless/pugilist/chaplain
+	name = "armwraps of unyielding resolve"
+	desc = "A series of armwraps, soaked in holy water. Makes you pretty keen to smite evil magic users."
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	enhancement = 1 //It is not magic that makes you punch harder, but force of will. Trust me.
+	secondary_trait = TRAIT_ANTIMAGIC
+	var/chaplain_spawnable = TRUE
```

Comment:
This variable is currently unused, right?

Reply:
So I'm entirely unsure how exactly the non-null rod code is working, but I believe it needs this var when the null rod list looks for stuff it can spawn.

Big shrug. I hadn't even tested this bit yet since I have a lot more functionality stuff to do first and I can't find any documentation as to how that stuff works.